<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppearancePreference</class>
 <widget class="QWidget" name="AppearancePreference">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Appearance</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="fontGroupBox">
     <property name="title">
      <string>Font Settings</string>
     </property>
     <layout class="QFormLayout" name="fontFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="fontFamilyLabel">
        <property name="text">
         <string>Font Family:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="fontFamilyComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="fontSizeLabel">
        <property name="text">
         <string>Font Size:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="fontSizeSpinBox">
        <property name="minimum">
         <number>6</number>
        </property>
        <property name="maximum">
         <number>72</number>
        </property>
        <property name="value">
         <number>12</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="fontPreviewLabel">
        <property name="text">
         <string>Preview:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="fontPreview">
        <property name="text">
         <string>Sample Text</string>
        </property>
        <property name="styleSheet">
         <string>border: 1px solid gray; padding: 8px; background-color: white;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="themeGroupBox">
     <property name="title">
      <string>Theme Settings</string>
     </property>
     <layout class="QFormLayout" name="themeFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="styleLabel">
        <property name="text">
         <string>Application Style:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="styleComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="colorThemeLabel">
        <property name="text">
         <string>Color Theme:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="colorThemeComboBox">
        <item>
         <property name="text">
          <string>System</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Light</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Dark</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="languageLabel">
        <property name="text">
         <string>Language:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="languageComboBox">
        <item>
         <property name="text">
          <string>English</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Chinese</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="previewGroupBox">
     <property name="title">
      <string>Preview</string>
     </property>
     <layout class="QVBoxLayout" name="previewLayout">
      <item>
       <widget class="QLabel" name="previewLabel">
        <property name="text">
         <string>Application will use these settings after restart</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="styleSheet">
         <string>color: #666;</string>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
